engine-tests/src/tests/sanity.rs

```diff
@@ -453,6 +453,33 @@ fn test_call_too_deep_error() {
+        hex::decode(code).unwrap()
+    };
+    let result = runner
+        .submit_with_signer(&mut signer, |nonce| {
+            aurora_engine_transactions::legacy::TransactionLegacy {
+                nonce,
+                gas_price: U256::zero(),
+                gas_limit: u64::MAX.into(),
+                to: None,
+                value: Wei::zero(),
+                data: constructor_code,
+            }
+        })
+        .unwrap();
+    match result.status {
```

Comment:
It looks more idiomatic, imo
```rust
    assert!(
        matches!(result.status, TransactionStatus::OutOfGas),
        "Unexpected status: {:?}",
        result.status
    );
```

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/641/commits/0fee121e9cb5cd515300978c7b39ede53c8b4869